packages/app-project/pages/[panoptesEnv]/[owner]/[project]/index.js

```diff
@@ -1,17 +1,20 @@
 import getDefaultPageProps from '@helpers/getDefaultPageProps'
 export { default } from '@screens/ProjectHomePage'
 import { serverSideTranslations } from 'next-i18next/serverSideTranslations'
+import fetchProjectPageTitles from '@helpers/fetchProjectPageTitles'
 
 export async function getStaticProps({ locale, params }) {
   const { notFound, props } = await getDefaultPageProps({ locale, params })
+  const { project } = props.initialState
+  project.about_pages = await fetchProjectPageTitles(project, params.panoptesEnv)
```

Comment:
If you add this to the default page props, then the banner will show up on all pages. Otherwise, I think the results link will be missing on `/classify`.

Reply:
Good catch! I'll put this fetch in `getStaticPageProps` insteaad